src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -204,4 +235,37 @@ private static function hasColorSupport()
 
         return defined('STDOUT') && function_exists('posix_isatty') && @posix_isatty(STDOUT);
     }
+
+    /**
+     * Useful when weak vendors mode is enabled.
+     *
+     * @param string $path absolute path to a source file
+     *
+     * @return bool
+     */
+    private static function inVendors($path)
+    {
+        if (self::$testMode) {
```

Comment:
When testing, `$path` equals `-`, and I had no idea how to change that, so I added a public property dedicated to tests, because [testing trumps encapsulation](https://8thlight.com/blog/uncle-bob/2015/06/30/the-little-singleton.html), says Uncle Bob.